ophyd/epics_motor.py

```diff
@@ -42,12 +48,23 @@ class EpicsMotor(Device, PositionerBase):
     settle_time : float, optional
         The amount of time to wait after moves to report status completion
     '''
+    user_offset = Cpt(EpicsSignal, '.OFF')
     user_readback = Cpt(EpicsSignalRO, '.RBV')
     user_setpoint = Cpt(EpicsSignal, '.VAL', limits=True)
     motor_egu = Cpt(EpicsSignal, '.EGU')
     motor_is_moving = Cpt(EpicsSignalRO, '.MOVN')
     motor_done_move = Cpt(EpicsSignalRO, '.DMOV')
     motor_stop = Cpt(EpicsSignal, '.STOP')
+    motor_offset_freeze_switch = Cpt(EpicsSignal, '.FOFF')
```

Comment:
`motor_egu` is a necessity, otherwise it clashes with the `egu` property (`'stop'` is similar, and `'is_moving'`/`'done_move'` sounds too much like existing methods). `namedtuple` makes it such that we can't name `motor_egu` `_egu` instead (no underscores in its keys). 

I'm very much in favor of removing the leading `motor_` where it won't cause confusion.